### ISSUE TITLE
loss=nan occurs during training

### ISSUE THREAD

**Issue by dongyyyy** [prs-eth/Marigold#111]
When I train with images of the same resolution, loss=0 occurs. It doesn't matter if I reduce my learning rate to one-tenth of what it was.
<img width="737" alt="Q" src="https://github.com/user-attachments/assets/de471af3-5e56-4270-b4f1-54207735f61c">

**Comment by YacineDeghaies** [comment]
@dongyyyy  did you solve this ?

**Comment by GaParmar** [comment]
I encountered the same problem. I notice that this happens when input batch contains NaN values.

**Comment by markkua** [comment]
Hi, I would suggest checking your data -- does it contain NaN in the input image or ground truth?

**Comment by dongyyyy** [comment]
> @dongyyyy did you solve this ?

Yes, thank you

**Comment by kke19** [comment]
@dongyyyy Hi, could you tell me how you solved this problem?
